### ISSUE TITLE
contact company names

### ISSUE THREAD

**Issue by punkyard** [nextcloud/maps#789]
hi, if a contact is just a company, without name and surname, flags happen to be nameless on the map ..

**Comment by tacruc** [comment]
@punkyard thank you for reporting the Issue. Could you provide some sample vCard, which I can use to reproduce the Issue?

**Comment by punkyard** [comment]
not really, i've just created 3 sample contacts in dev .. to give it a try

**Comment by punkyard** [comment]
could you consider a list of different icons for the favorites ? could make the map more readable
let's say i have a group of favorites for gaz station, another for hospitals, different icons on the map according to the group of favorites would be a blaze ..

**Comment by punkyard** [comment]
hi
is there any advancement in this matter?
